### PR TITLE
Add org-wide code of conduct

### PR DESCRIPTION
**Pull request by nslee333**
Starting a PR for a code of conduct - one of the tasks in issue #79 in alveusgg repo.

### CODE REVIEW

**Comment by nslee333** [comment]
What do you guys think of just using the contributor covenant as suggested by @pjeweb? @MattIPv4

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+
```
Comment (MattIPv4) [inline CODE_OF_CONDUCT.md]:
```suggestion
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+opensource@alveussanctuary.org.
```
Comment (MattIPv4) [inline CODE_OF_CONDUCT.md]:
```suggestion
[opensource@alveussanctuary.org](mailto:opensource@alveussanctuary.org).
```

**Review by MattIPv4 — COMMENTED** [review]
Using Contributor Covenant seems fine to me, it is rather standard and well adopted in the space :)

**Review by MattIPv4 — APPROVED** [review]

**Review by pjeweb — APPROVED** [review]

**Pull request merged**